extensions/src/main/java/org/pentaho/platform/web/http/context/PentahoSystemReadyListener.java

```diff
@@ -64,6 +65,17 @@ public void contextInitialized( ServletContextEvent servletContextEvent ) {
 
   @Override
   public void contextDestroyed( ServletContextEvent servletContextEvent ) {
+    Repository jcrRepository = PentahoSystem.get( Repository.class, "jcrRepository", null );
+    if ( jcrRepository == null ) {
+      System.err.println( "Cannot obtain JCR repository. Exiting" );
+      return;
+    }
+    if ( !( jcrRepository instanceof JackrabbitRepository ) ) {
+      System.err.println(
```

Comment:
Sorry I didn't see this sooner - the class already has a Logger - please use log.error here.